### PR TITLE
Fix tunerstudio ratio vs. percent

### PR DESCRIPTION
**Pull request by mck1117**
There was a mystery mix of:
- ratio (example 1.2)
- percent (example 120%)
- percent extra (example +20%)

These have all been consolidated down to use only the "ratio" option.

### CODE REVIEW

**Comment by rusefi** [comment]
Can one of the admins verify this patch?

**Comment by mck1117** [comment]
This commit includes a revert-ish of be926cb.  That commit took the numbers being spit out as canon, and as a result, missed the fact that cranking fuel pulse widths were 10x what they're supposed to be.  This change both fixes the actual bug (wrong cranking clt multiplier) and the tests.

File: firmware/controllers/algo/engine_configuration.cpp [hunk 1]
```diff
@@ -632,11 +632,11 @@ static void setDefaultCrankingSettings(DECLARE_ENGINE_PARAMETER_SIGNATURE) {
...
 	engineConfiguration->crankingAdvanceBins[0] = 0;
 	engineConfiguration->crankingAdvance[1] = 0;
-	engineConfiguration->crankingAdvanceBins[0] = 200;
+	engineConfiguration->crankingAdvanceBins[1] = 200;
 	engineConfiguration->crankingAdvance[2] = 0;
-	engineConfiguration->crankingAdvanceBins[0] = 400;
+	engineConfiguration->crankingAdvanceBins[2] = 400;
 	engineConfiguration->crankingAdvance[3] = 0;
-	engineConfiguration->crankingAdvanceBins[0] = 1000;
+	engineConfiguration->crankingAdvanceBins[3] = 1000;
```
Comment (mck1117) [inline firmware/controllers/algo/engine_configuration.cpp]: These were just at typo.

**Comment by mck1117** [comment]
Oh, and this should close #773.

**Pull request merged**